components/ui-api-layer/internal/domain/remoteenvironment/re_converter.go

```diff
@@ -30,26 +30,21 @@ func (c *remoteEnvironmentConverter) ToGQL(in *v1alpha1.RemoteEnvironment) gqlsc
 	dto := gqlschema.RemoteEnvironment{
 		Name:        in.Name,
 		Description: in.Spec.Description,
-		Source: gqlschema.RemoteEnvironmentSource{
```

Comment:
was it done on purpose, removing resource filed from graphql (and not including sourceId)

Reply:
In adittion, shouldn't we also add `Labels JSON`?

Reply:
It's done on purpose. Now only EventActivation has SourceId filed